in_toto/model.go

```diff
@@ -112,6 +127,31 @@ type Step struct {
 	SupplyChainItem
 }
 
+func (s *Step) validate() {
+	validateFunctions := []func(){s.validateType, s.validatePubkeys}
+	for _, f := range validateFunctions {
+		f()
+	}
+}
+
+func (s *Step) validateType() {
+	if s.Type != "step" {
+		fmt.Println("Invalid Type value for step: should be 'step'")
+	}
+}
+
+func (s *Step) validatePubkeys() {
+	for _, keyId := range s.PubKeys {
+		keyIdFormatCheck, err := regexp.MatchString("[a-fA-F0-9]+", keyId)
```

Comment:
I think this should be a separate key id validator, to be re-used with e.g. a `Key` struct validator.